src/libraries/Microsoft.PowerFx.Core/Utils/FormulaSort/IdentifierVisitorScope.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.PowerFx.Core.Utils.FormulaSort
+{
+    internal sealed class IdentifierVisitorScope
+    {
+        private readonly Dictionary<string, IdentifierVisitorMode> _idents = new Dictionary<string, IdentifierVisitorMode>();
```

Comment:
>string [](http://example.com/codeflow?start=36&length=6)

nit: can this be DName? avoids the casing questions.

Reply:
Removed, this PR can focus just on Topsort